core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorPartitioningHandle.java

```diff
@@ -24,4 +24,10 @@ default boolean isCoordinatorOnly()
     {
         return false;
     }
+
+    // It helps the engine to identify whether writer scaling per partition is allowed
```

Comment:
`It helps the engine to identify whether writer scaling per partition is allowed`
->
`Specifies if writing to partition has to be performed by a single writer instance`